invenio_app_rdm/records_ui/views/records.py

```diff
@@ -140,19 +141,38 @@ def record_detail(pid_value, record, files, is_preview=False):
         emitter(current_app, record=record._record, via_api=False)
 
     resolved_community, _ = get_record_community(record_ui)
+
+    try:
+        externalDOI = (
+            not current_app.config["RDM_LOCK_FILES_FOR_EXTERNAL_DOI"]
+            and dict_lookup(record, "pids.doi.provider")
+            == current_app.config["RDM_EXTERNAL_DOI_PROVIDER_KEY"]
+        )
+    except KeyError:
+        externalDOI = False
+
     return render_template(
         current_app.config.get("APP_RDM_RECORD_LANDING_PAGE_TEMPLATE"),
         record=record_ui,
         files=files_dict,
         user_communities_memberships=get_user_communities_memberships(),
         permissions=record.has_permissions_to(
-            ["edit", "new_version", "manage", "update_draft", "read_files", "review"]
+            [
+                "edit",
+                "new_version",
+                "manage",
+                "update_draft",
+                "read_files",
+                "review",
+                # "draft_modify_files",
```

Comment:
I'd call this "show_version_button". Though I'd check what the "new_version" permission does to see if that might be duplicative.